services/staging/rs_server_staging/main.py

```diff
@@ -191,53 +288,42 @@ async def execute_process(req: Request, resource: str, data: ProcessMetadataMode
 @router.get("/jobs/{job_id}")
 async def get_job_status(job_id: str = Path(..., title="The ID of the job")):
     """Used to get status of processing job."""
-    with app.extra["db_handler"]:
-        job = app.extra["db_table"].get(Query().job_id == job_id)
-
-        if job:
-            return job
-
-        raise HTTPException(status_code=HTTP_404_NOT_FOUND, detail="Job not found")
+    job = app.extra["process_manager"].get_job(job_id)
+    if job:
+        return job
+    raise HTTPException(status_code=404, detail="Job not found")
 
 
 @router.get("/jobs")
 async def get_jobs():
     """Returns the status of all jobs."""
-    with app.extra["db_handler"]:
-        jobs = app.extra["db_table"].all()  # Retrieve all job entries from the jobs table
+    jobs = app.extra["process_manager"].get_jobs()
 
-        if jobs:
-            return JSONResponse(status_code=HTTP_200_OK, content=jobs)
+    if jobs:
+        return JSONResponse(status_code=HTTP_200_OK, content=jobs)
 
-        # If no jobs are found, return 404 with appropriate message
-        raise HTTPException(status_code=HTTP_404_NOT_FOUND, detail="No jobs found")
+    # If no jobs are found, return 404 with appropriate message
+    raise HTTPException(status_code=HTTP_404_NOT_FOUND, detail="No job found")
 
 
 @router.delete("/jobs/{job_id}")
 async def delete_job(job_id: str = Path(..., title="The ID of the job to delete")):
```

Comment:
Maybe change endpoint function name, since is shadowing the one from process_manager

Reply:
I changed them